ansible/roles/shared_dir/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+#include: ../../shared_data_dir/vars/main.yml
```

Comment:
Commented out because this file does not appear to do anything, but this points to the place where the relevant vars can be found. See also https://github.com/dimagi/commcarehq-ansible/blob/852037964c8528edcb19ec9e60e6e4f00386c5fb/ansible/roles/shared_dir.yml#L7